site/app/templates/admin/admin_gradeable/AdminGradeablePeers.twig

```diff
@@ -1,10 +1,37 @@
-<fieldset>
-    <legend>Upload a csv file dictating grader-student pairs for peer grading. The csv file must contain a labelled 'student' column and a 'grader' column:</legend>
-    <label for="peer_graders_list">Select CSV file to upload for grader-student pairs</label>
-    <input type="file" name="peer_graders_list" id="peer_graders_list">
-
-    </br>
-     <div id="peer_loader" class="loader hide"></div>
+<div id="peer-control">
+    <h1> Options for Peer Matrix </h1>
+    <div class="row">
+    <div class="card col-md-6" style="margin:0px">
+        <div class="content" style="margin:0px">Upload a csv file dictating grader-student pairs for peer grading. The csv file must contain a labelled 'student' column and a 'grader' column: <BR><hr>
+        <input class="custom-file-input" type="file" name="peer_graders_list" id="peer_graders_list"> </div>
+    </div>
+    <div class="card col-md-6" style="margin:0px">
+        <div class="content" style="margin:0px">Enter number of peers for one student to grade and randomly assign peer grading in class. <br>
+            <hr>
+            <input type="checkbox" class="ignore" name="restrict_to_registration" id="restrict_to_registration" value="Restrict to Registration Sections">Restrict to Registration Sections</input>
```

Comment:
Lets use dash's instead of underscores for CSS id's to be consistent with the rest of submitty
e.g: `restrict-to-registration`

Reply:
Done!

Reply:
corrected!